docs/constraint_creation.md

```diff
@@ -76,6 +76,17 @@ pod_has_hostipc {
 
 The comment block is also what is used when generating documentation via the `doc` command.
 
+### Annotating rules for matchers
+
+- `@kinds` generates `constraint.spec.match.kinds`. The allowed format is whitespace-separated `<group>/<kind>` list.
```

Comment:
nit: I would prefer `whitespace` changed to `space` to clarify we don't support tabs.

Reply:
Certainly it looks better so I fixed it